2021-05-06 11:32:12,692 INFO    MainThread:10008 [wandb_setup.py:_flush():69] setting env: {}
2021-05-06 11:32:12,692 INFO    MainThread:10008 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-06 11:32:12,693 INFO    MainThread:10008 [wandb_init.py:_log_setup():336] Logging user logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210506_113212-347qaknc\logs\debug.log
2021-05-06 11:32:12,693 INFO    MainThread:10008 [wandb_init.py:_log_setup():337] Logging internal logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210506_113212-347qaknc\logs\debug-internal.log
2021-05-06 11:32:12,694 INFO    MainThread:10008 [wandb_init.py:init():369] calling init triggers
2021-05-06 11:32:12,695 INFO    MainThread:10008 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'epochs': 25, 'learning_rate': 0.0001, 'batch_size': 128, 'input_dim': 256, 'output_dim': 4, 'fc_size_1': 128, 'fc_size_2': 64, 'fc_size_3': 32, 'fc_size_4': 16}
2021-05-06 11:32:12,695 INFO    MainThread:10008 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 1q0i5842
2021-05-06 11:32:12,697 INFO    MainThread:10008 [wandb_run.py:finish():1197] finishing run stevenpach10/MLP/1q0i5842
2021-05-06 11:32:12,700 ERROR   MainThread:10008 [jupyter.py:save_history():312] Run pip install nbformat to save notebook history
2021-05-06 11:32:12,701 INFO    MainThread:10008 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-06 11:32:12,701 INFO    MainThread:10008 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-06 11:32:12,701 INFO    MainThread:10008 [wandb_run.py:_atexit_cleanup():1524] got exitcode: 0
2021-05-06 11:32:12,702 INFO    MainThread:10008 [wandb_run.py:_restore():1496] restore
2021-05-06 11:32:13,838 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1100
  total_bytes: 1100
}

2021-05-06 11:32:14,640 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1100
  total_bytes: 3152
}

2021-05-06 11:32:15,051 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1100
  total_bytes: 3154
}

2021-05-06 11:32:15,169 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3154
  total_bytes: 3154
}

2021-05-06 11:32:15,277 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3154
  total_bytes: 3154
}

2021-05-06 11:32:15,387 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3154
  total_bytes: 3154
}

2021-05-06 11:32:15,499 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3154
  total_bytes: 3154
}

2021-05-06 11:32:15,608 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3154
  total_bytes: 3154
}

2021-05-06 11:32:15,712 INFO    MainThread:10008 [wandb_run.py:_wait_for_finish():1646] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3154
  total_bytes: 3154
}

2021-05-06 11:32:16,863 INFO    MainThread:10008 [wandb_run.py:_show_files():1868] logging synced files
2021-05-06 11:32:16,874 INFO    MainThread:10008 [wandb_init.py:init():418] starting backend
2021-05-06 11:32:16,874 INFO    MainThread:10008 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-05-06 11:32:16,881 INFO    MainThread:10008 [backend.py:ensure_launched():123] starting backend process...
2021-05-06 11:32:17,012 INFO    MainThread:10008 [backend.py:ensure_launched():127] started backend process with pid: 14108
2021-05-06 11:32:17,013 INFO    MainThread:10008 [wandb_init.py:init():423] backend started and connected
2021-05-06 11:32:17,016 INFO    MainThread:10008 [wandb_init.py:init():465] updated telemetry
2021-05-06 11:32:17,017 INFO    MainThread:10008 [wandb_init.py:init():484] communicating current version
2021-05-06 11:32:18,513 INFO    MainThread:10008 [wandb_init.py:init():489] got version response 
2021-05-06 11:32:18,513 INFO    MainThread:10008 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-06 11:32:18,951 INFO    MainThread:10008 [wandb_init.py:init():522] starting run threads in backend
2021-05-06 11:32:19,990 INFO    MainThread:10008 [wandb_run.py:_console_start():1554] atexit reg
2021-05-06 11:32:19,992 INFO    MainThread:10008 [wandb_run.py:_redirect():1401] redirect: SettingsConsole.WRAP
2021-05-06 11:32:19,992 INFO    MainThread:10008 [wandb_run.py:_redirect():1444] Wrapping output streams.
2021-05-06 11:32:19,992 INFO    MainThread:10008 [wandb_run.py:_redirect():1468] Redirects installed.
2021-05-06 11:32:19,992 INFO    MainThread:10008 [wandb_init.py:init():546] run started, returning control to user process
2021-05-06 11:32:19,993 INFO    MainThread:10008 [wandb_watch.py:watch():39] Watching
2021-05-06 11:32:46,867 INFO    MainThread:10008 [wandb_run.py:finish():1197] finishing run stevenpach10/MLP/347qaknc
